griptape/loaders/email_loader.py

```diff
@@ -11,7 +11,7 @@
 
 
 @define
-class EmailLoader(BaseLoader):
+class EmailLoader(BaseLoader["EmailLoader.EmailQuery", list[bytes], ListArtifact]):  # pyright: ignore[reportGeneralTypeIssues]
     @define(frozen=True)
     class EmailQuery:
```

Comment:
I know we like the nested classes, but wouldn't it be easier to just allow related classes in the same file?

The choice for nested classes was really about style after all wasn't it?